### PR TITLE
test(patients-search): convert to RTL

### PR DESCRIPTION
**Pull request by kiranjd**
First PR

### CODE REVIEW

File: src/__tests__/patients/search/NoPatientsExist.test.tsx [hunk 1]
```diff
@@ -1,33 +1,24 @@
...
 
 import NoPatientsExist from '../../../patients/search/NoPatientsExist'
 
 describe('NoPatientsExist', () => {
-  const setup = () => mount(<NoPatientsExist />)
+  const render = () => rtlRender(<NoPatientsExist />)
 
   it('should render an icon and a button with typography', () => {
-    const wrapper = setup()
+    render()
```
Comment (mpeyper) [inline src/__tests__/patients/search/NoPatientsExist.test.tsx]: For the one time this is is called, I'd just inline it:

```suggestion
import { render, screen } from '@testing-library/react'
import React from 'react'

import NoPatientsExist from '../../../patients/search/NoPatientsExist'

describe('NoPatientsExist', () => {
  it('should render an icon and a button with typography', () => {
    render(<NoPatientsExist />)
```

File: src/__tests__/patients/search/ViewPatients.test.tsx [hunk 2]
```diff
@@ -33,8 +34,8 @@ describe('Patients', () => {
   })
 
   it('should render the search patients component', () => {
-    const wrapper = setup()
+    render()
```
Comment (mpeyper) [inline src/__tests__/patients/search/ViewPatients.test.tsx:36]: There's too many gaps to post a suggested change, but again, I would inline this into the test.

**Review by mpeyper — APPROVED** [review]
In general, I'm not a fan of the `render` helpers and renaming the `rtlRender` import.   If the helper is something we want keep, consider renaming it `renderComponent` and keen the `@testing-library/react` import as just `render`.

Just suggestions though.  The actual changes here are fine.

**EDIT:** I see the other PRs are doing this as well, so maybe my suggestions are off base.  I stand by my comments but happy to fall in line with the consensus.

File: src/__tests__/patients/search/NoPatientsExist.test.tsx (hunk 1 above)
Comment (kiranjd) [reply src/__tests__/patients/search/NoPatientsExist.test.tsx]: This makes sense. Making this change

File: src/__tests__/patients/search/ViewPatients.test.tsx (hunk 2 above)
Comment (kiranjd) [reply src/__tests__/patients/search/ViewPatients.test.tsx:36]: I prefer to keep it this way if it's okay. This component seems like there is scope for changes. Removed the extra assignment to render function though.

**Comment by kiranjd** [comment]
Closing this PR. Messed my rebase and somehow this PR is addressing the last 12 commits instead of 2.

**Pull request closed without merging**